src/schema.rs

```diff
@@ -450,41 +450,48 @@ fn overwrite_trycmd_status(
     line_nums: &mut std::ops::Range<usize>,
     normalized: &mut String,
 ) -> Result<(), snapbox::Error> {
-    if let Some(status) = exit {
-        if let Some(status) = if let Some(code) = status.code() {
-            if status.success() {
-                if let (true, Some(line_num)) = (
-                    expected_status != Some(CommandStatus::Success),
-                    expected_status_source,
-                ) {
-                    replace_lines(normalized, line_num..line_num + 1, "")?;
-                    *line_nums = line_nums.start - 1..line_nums.end - 1;
-                }
-                None
-            } else {
-                match expected_status {
-                    Some(CommandStatus::Success | CommandStatus::Interrupted) => {
-                        Some(format!("? {code}"))
-                    }
-                    Some(CommandStatus::Code(expected)) if expected != code => {
-                        Some(format!("? {code}"))
-                    }
-                    _ => None,
-                }
+    let status = match exit {
+        Some(status) => status,
+        _ => {
+            return Ok(());
+        }
+    };
```

Comment:
Side note: when we upgrade MSRV, this can be simplified with let-else

Reply:
yep that's what I was originally using and got hit by the CI

Reply:
Recently got to use it in another project and it was really handy: https://github.com/gitext-rs/git2-ext/blob/55616965732cc4ff6d63191bbd226b530e8c7e59/src/ops.rs#L634,L647

Reply:
Was able to use that in another project too: https://github.com/nix-community/patsh/commit/d7740b8f39677b6e8a73af38da8debcdf93babcc, makes stuff a lot more readable